sigs.yaml

```diff
@@ -2306,6 +2306,39 @@ workinggroups:
     teams:
     - name: wg-component-standard
       description: Component Standard Discussion
+- dir: wg-data-protection
+  name: Data Protection
+  mission_statement: >
+    A Working Group dedicated to promoting data protection support in Kubernetes,
+    identifying missing functionality and working together to design features to enable
+    data protection support. Involves collaboration with multiple SIGs such as Apps
+    and Storage.
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Apps
+  - Storage
```

Comment:
I think Auth should be added here as well.

Reply:
Will do.  Thanks!